### PR TITLE
chore: update READ.md

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- Update documentation for gitbook integration
- Moves directory structure to root of every package
- Add README for thegraph
- Update core readme which links to packages 
- remove networks.json (redundant file)

https://app.gitbook.com/o/Aqbtj6s4OkLaygileCka/s/6WA8Y0HFz1HDvz5JgLH8/

### CODE REVIEW

**Review by chibie — CHANGES_REQUESTED** [review]
lgtm! few typos

File: packages/round-manager/README.md [hunk 1]
```diff
@@ -1,62 +1,51 @@
 # round-manager
 
-This package serves the app which holds all the features w.r.t to managing a round.
+This package serves the app which holds all the features w.r.t to 
 
-## Usage
+- creating a program
+- maintaing a program
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply packages/round-manager/README.md:6]:
```suggestion
- maintaining a program
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,86 @@
...
 
 ## Directory Structure
+
 ```
 .
 ├── .github                     # github specific configuration
 ├── packages
 │   ├── contracts               # contracts built using hardhat
-│   ├── round-manager           # react-frontend for round-mamager
+│   ├── graph                   # graph which indexes data from contracts
```
Comment (Abuchtela) [inline README.md:18]: `├── graph # graph which indexes data from contracts`

Comment (Abuchtela) [reply README.md:18]:
```suggestion
│   ├── graph                   # graph which indexes data from contracts
```

**Pull request merged**